pkg/apis/cni/v1beta1/cni.pb.go

```diff
@@ -1,29 +1,43 @@
+// Copyright 2019 Antrea Authors
```

Comment:
Are file changes of cni related to Kafka flow export support for flow aggregator? If not, I feel this should be put in a separate PR.

Reply:
This is because of version change in protoc-gen-go. All this is generated code.

Reply:
I agree with @zyiou (left another comment above). The Protobuf version upgrade should be handled in a separate PR, even if the upgrade is only motivated by this change.